src/components/date-picker/DatePicker.vue

```diff
@@ -30,9 +30,9 @@
             :clearable="clearable"
             :error-messages="errorMsg"
             :error="!!errorMsg"
-            :value="displayDate"
+            :model-value="displayDate"
```

Comment:
This is a two-way binding now, right? So does displayDate get updated when the user types in something?

Reply:
It doesn't update since the field is readonly. The picker wasn't updating when it was `value`, I think because in Vue 3 they changed that prop to `model-value`: https://v3-migration.vuejs.org/breaking-changes/v-model.html

Reply:
The way I've seen it done in a couple of Ketaki's tickets is to use kebab-case, I think because we generally use it for our props already